video_sampler/gating.py

```diff
@@ -13,7 +13,11 @@
     import open_clip
     import torch
 
-    DEVICE = "cuda" if torch.cuda.is_available() else "cpu"
+    DEVICE = "cpu"
+    if torch.backends.cuda.is_built():
```

Comment:
Maybe `is_available` vs `is_built` causes the issue with tests on the cpu machines.